### PR TITLE
Fixed bug where parent fields could not be merged before saving.

### PR DESCRIPTION
**Pull request by donggyu04**
1. Add last modified date and user when init user table.
2. Fixed bug where parent fields could not be merged before saving.
     - `getClass().getDeclaredFields()` could not resolve parent fields.

### CODE REVIEW

File: ngrinder-controller/src/main/java/org/ngrinder/infra/init/DBInit.java [hunk 1]
```diff
@@ -100,7 +100,12 @@ private void createUser(String userId, String password, Role role, String userNa
 			user.setRole(role);
 			user.setUserName(userName);
 			user.setEmail(email);
-			user.setCreatedDate(new Date());
+
+			Date createdDate = new Date();
```
Comment (junoyoon) [inline ngrinder-controller/src/main/java/org/ngrinder/infra/init/DBInit.java]: ```Date now = new Date()```
might be better name for it.

File: ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java [hunk 2]
```diff
@@ -77,4 +80,15 @@ private static void makeAccessible(final Field field) {
 		}
 	}
 
+	public static List<Field> getDeclaredFieldsUpToParent(Class<?> currentClass, Class<?> exclusiveParent) {
```
Comment (junoyoon) [inline ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: What is ```exclusiveParent``` ?
Is it really necessary.

And just ```getDeclaredFieldsIncludingParent``` might be  better name.

* currentClass => current
* parentClass => parent

File: ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java [hunk 3]
```diff
@@ -77,4 +80,15 @@ private static void makeAccessible(final Field field) {
...
+		List<Field> currentClassFields = newArrayList(currentClass.getDeclaredFields());
+		Class<?> parentClass = currentClass.getSuperclass();
+
+		if (parentClass != null && !(parentClass.equals(exclusiveParent))) {
+			List<Field> parentClassFields = getDeclaredFieldsUpToParent(parentClass, exclusiveParent);
```
Comment (junoyoon) [inline ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: Is should be recursive call?

File: ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java [hunk 4]
```diff
@@ -77,4 +80,15 @@ private static void makeAccessible(final Field field) {
 		}
 	}
 
+	public static List<Field> getDeclaredFieldsUpToParent(Class<?> currentClass, Class<?> exclusiveParent) {
+		List<Field> currentClassFields = newArrayList(currentClass.getDeclaredFields());
```
Comment (junoyoon) [inline ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: I think fields is better name than currentClassFields

File: ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java (hunk 2 above)
Comment (donggyu04) [reply ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: It is optional.
ngirnder may be not use this.

is better remove this?

File: ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java (hunk 3 above)
Comment (donggyu04) [reply ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: If exist function that get all parent class, i can use loop instead of recursive.
i'll check this.

Comment (junoyoon) [reply ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: In java, a class allow only one parent class so we don't need recursive call.. loop is enough.

File: ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java [hunk 5]
```diff
@@ -33,6 +38,7 @@
 public abstract class ReflectionUtils {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(ReflectionUtils.class);
+	private static Map<Class, List<Field>> fieldCache = new ConcurrentHashMap<>();
```
Comment (junoyoon) [inline ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: please put final.

File: ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java [hunk 6]
```diff
@@ -77,4 +83,23 @@ private static void makeAccessible(final Field field) {
 		}
 	}
 
+	public static List<Field> getDeclaredFieldsIncludingParent(Class<?> current) {
+
+		if (fieldCache.containsKey(current)) {
+			return fieldCache.get(current);
+		}
+
+		Class<?> start = current;
```
Comment (junoyoon) [inline ngrinder-core/src/main/java/org/ngrinder/common/util/ReflectionUtils.java]: why is this necessary?
Isn't better to fieldCache.put below here?

**Comment by junoyoon** [comment]
LGTM
